src/app/httpapi.js

```diff
@@ -479,13 +479,13 @@
 
             server.expose('getsorters', function (args, opt, callback) {
                 switch (App.currentview) {
-                case 'shows':
+                case 'tvshow':
                 case 'anime':
                     butterCallback(callback, false, {
                         'sorters': App.Config.sorters_tv
                     });
                     break;
-                case 'movies':
+                case 'movie':
```

Comment:
not related to this PR, i'm just fixing what needs fixing, I'm not rewriting something else than list. see #576